DevelopGitHub/GitHubExtension/GitHubExtension.EntryPoint/Startup.cs

```diff
@@ -46,6 +46,7 @@ public void Configuration(IAppBuilder app)
 
             ConfigureOauth(app);
             ConfigureCookies(app);
+            app.Use<TokenCheckMiddleware>();
```

Comment:
So as I understand you need this middleware only for inbound HTTP requests to verify claims?